public/app/plugins/datasource/jaeger/_importedDependencies/types/TTraceDiffState.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
Does not seem to be used anywhere, here or in the TraceView. Think I removed this in another PR.